lib/cluster.js

```diff
@@ -165,7 +165,7 @@ function addListeners() {
 	if (listenersAdded) return;
 	listenersAdded = true;
 
-	if (cluster().isMaster) {
+	if (cluster().isPrimary) {
```

Comment:
I like this change, but should it be in this PR?

Reply:
I think it’s also in another PR that you just merged.  Let me rebase and see how much of this evaporates.

Reply:
Confirmed, this is from #680, but I needed the benchmark changes I made to make parts of that PR testable, and that was buried in the middle and I’ve been doing too much interactive rebasing on this branches as it is.

Reply:
@SimenB After rebasing, #683 has only 4 commits and all of them are also in this PR. I did not remember doing that. Maybe I was touching the same code and didn't want merge conflicts?

(I just fixed a couple in this one where I ended up having to touch lines that either were in the previous request or one line below, which annoys git)